### PR TITLE
Requesting test EC2 instance for NF gene drug DB.

### PR DESCRIPTION
**Pull request by milen-sage**

### CODE REVIEW

File: config/prod/gene_drug_db_test_instance.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2-v2.yaml
+stack_name: gene_drug_db_test_instance
+parameters:
+  # The Sage deparment for this resource
+  Department: "Platform"
```
Comment (zaro0508) [inline config/prod/gene_drug_db_test_instance.yaml]: is this your Department and Project?

File: config/prod/gene_drug_db_test_instance.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2-v2.yaml
+stack_name: gene_drug_db_test_instance
```
Comment (zaro0508) [inline config/prod/gene_drug_db_test_instance.yaml]: A stack name can contain only alphanumeric characters (case-sensitive) and hyphens.  Also would you mind adding `-ec2` to the end of the stack name and to the corresponding filename?

Comment (milen-sage) [reply config/prod/gene_drug_db_test_instance.yaml]: 👍 This also should be fixed now.

File: config/prod/gene_drug_db_test_instance.yaml (hunk 1 above)
Comment (milen-sage) [reply config/prod/gene_drug_db_test_instance.yaml]: That makes sense :) 👍 It should be fixed with my new commit.

**Pull request merged**